ember_debug/models/profile-manager.js

```diff
@@ -5,6 +5,74 @@ const {
   run: { later, scheduleOnce },
 } = Ember;
 
+function getUnfilteredRoots(first, last, closest) {
+  const roots = [];
+  let start = null;
+  let end = null;
+  for (let i = 0; i < closest.length; i++) {
+    if (closest.item(i) === first.node) start = i;
+    else if (closest.item(i) === last.node) end = i;
+  }
+
+  if (start === null || end === null) return [];
+
+  for (let i = start; i <= end; i++) roots.push(closest.item(i));
+
+  return roots;
+}
+
+function _findRoots({ first, last, parent }) {
+  const closest = parent.childNodes;
+  if (first.node === last.node) return [first.node];
+
+  const roots = getUnfilteredRoots(first, last, closest);
+
+  return roots.filter((el) => el.nodeType === 1);
+}
+
+function makeHighlight() {
+  const node = document.createElement('div');
+  node.setAttribute('role', 'presentation');
+  node.setAttribute('class', 'ember-inspector-render-highlight');
+  return node;
+}
+function _insertHTML() {
+  return document.body.appendChild(makeHighlight());
+}
+
+function _insertStylesheet() {
+  const content = `
+    .ember-inspector-render-highlight {
+      border: 1px solid red;
```

Comment:
for multiple rerender cases (to get it more visible), I assume we could have box-shadow with .5 opacity (to allow layers multiply)

Reply:
changed to
`border: 2px solid rgba(255,0,0,0.2);
      box-shadow: 0px 0px 1px rgba(255,0,0,0.2);`